helm/helm-scpts-v3/installExporterChart

```diff
@@ -46,23 +43,16 @@ do
   fi
 done
 
-tls='--tls'
-enabled='N'
-tlsStatus=$(helm ls 2>&1)
-if [[ "$tlsStatus" != "Error: transport is closing" ]]; then
-    read -p 'Warning: Helm TLS is not enabled. Do you want to continue? [y/N] ' enabled
-    if [ "$enabled" != "y" ]
-    then
-        echo "Setup helm TLS. Follow https://helm.sh/docs/tiller_ssl/"
-        exit 1
-    fi
-fi
-
 nsCreateCmd=$(kubectl create namespace sysflow 2>&1)
 if [[ "$nsCreateCmd" =~ "already exists" ]]; then
-    echo "Warning: Namespace 'sysflow' already exists. Proceeding with existing namespace."
+    echo "Warning: Namespace 'sysflow' already exists. Proceeding with existing namespace"
 else
     echo "Namespace 'sysflow' created successfully"
 fi
 
-helm install sf-exporter-chart -f sf-exporter-chart/values.yaml.local --namespace sysflow --set sfexporter.s3AccessKey=$s3AccessKey --set sfexporter.s3SecretKey=$s3SecretKey --set sfexporter.s3Endpoint=$s3Endpoint --set sfexporter.s3Location=$s3Region --debug ./sf-exporter-chart
+# TODO: publish sysflow charts to https://kubernetes-charts.storage.googleapis.com,
+#       which is enabled by default, and get rid of below version and repo strings.
+chartVersion="0.1-rc2"
+chartRepo=https://weii666.github.io/charts/
```

Comment:
@weii666 did you mean to hardcode your personal repo here? Can you please fix this with the sf-deployments charts URL?

Reply:
This URL must be a Helm charts repo, where is index.yaml located.
My github is configured as a web page for this purpose.
We can make /sf-deployments/ be the Helm charts repo.
It needs to enable GitHub Pages in repo setting.

The correct way is to publish sysflow charts on https://kubernetes-charts.storage.googleapis.com as the TODO states above.
I can help to publish it.
Thank you.

Reply:
if we are using Operator installation on k8s, and let operator points to the charts served by ourselves, do we still need this helm-chart based installation?

Reply:
We still want to support vanilla k8's no?

Reply:
Publishing to helm repo is for community users.
It provides an option to deploy sysflow without operator.
The PR sent to helm is under review and has not got response.
I will try to ping helm members to continue.

Reply:
I got e2e test result from helm community.
Here is an error installing Sysdig:

I0414 12:19:07.157] Your kernel headers for kernel 4.19.104+ cannot be found at
......
I0414 12:19:07.158] E0414 12:18:59.790402    10 main.cpp:233] Runtime exception caught in main loop: error opening device /host/dev/sysdig0. Make sure you have root credentials and that the sysdig-probe module is loaded.

We do require privilege to mount /dev/ for sf-collector, 
which is not a normal behavior to any automatic tests.
I suspect we will have the same issue to pass RedHat certification.

Reply:
Looks like the issue is that you don't have the kernel headers installed on the host.

Reply:
Here is the helm PR and test links are within:
https://github.com/helm/charts/pull/21730